fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1508,9 +1508,15 @@ public class Config extends ConfigBase {
     public static int quorom_publish_wait_time_ms = 5000;
 
     /**
-     * Fqdn function switch, 
+     * Fqdn function switch,
      * this switch will be deleted after release the fqdn func
      */
     @ConfField(mutable = true)
     public static boolean enable_fqdn_func = false;
+
+    /**
+     * jaeger tracing endpoint, empty thing disables tracing
+     */
+    @ConfField
+    public static String jaeger_grpc_endpoint = "";
```

Comment:
Can the jaeger trace the memory consumption at the peak?
If we configure the endpoint, does it indicate we push the trace log to the jaeger?

Reply:
> Can the jaeger trace the memory consumption at the peak?
No, but if you can calculate the memory consumption at peak, you can add it to the span properties

> If we configure the endpoint, does it indicate we push the trace log to the jaeger?
Yes, the span data is in thrfit/protobuf format, and will be push to jaeger agent/server through udp/http/http2